src/use-localstorage.ts

```diff
@@ -4,7 +4,8 @@ import {
   LocalStorageChanged,
   isTypeOfLocalStorageChanged,
 } from './local-storage-events';
-import { useEffect, useState, useCallback } from 'react';
+import { storage  } from './storage'
+import { useEffect, useState, useCallback, useRef } from 'react';
```

Comment:
```suggestion
import { useEffect, useState, useCallback } from 'react';
```

`useRef` seems to be unused here 👀

Reply:
good spot!